docs/kbs.yaml

```diff
@@ -0,0 +1,267 @@
+openapi: 3.0.1
+info:
+  title: Key Broker Service (KBS) API
+  description: HTTP based API for describing the communication between a a
+               Key Broker Client in a confidential guest and a trusted
+               Key Broker Service.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.0
```

Comment:
I wonder if it's worth moving `version` up above `license:` to avoid any confusion with it being `license.version` ;)